script/Common/00_00.txt

```diff
@@ -739,22 +768,24 @@ That was probably why.
 [sha:d43f23018f3c49823d501abc85c21704da8e8660]{
 -- Page 52, Offset 254.
 -- あれから二週間も経つのに、誰も僕を信じてくれない。
-It had been two weeks since then and no one believed me.
+Even though two weeks had passed,
+ nobody had believed me.
 }
 [sha:8d6b8b0fee5f0969ca511743a359497a78d2d191]{
 -- Page 52, Offset 255.
 -- あれから二週間も経つのに、誰も僕に会いに来ない。
-It had been two weeks since then and no one had come to see me.
+Even though two weeks had passed,
+ nobody had come to see me.
```

Comment:
this is "in the past two weeks, nobody has come to see me" right? The way this is phrased now, it makes it sound like "nobody has come to see me at all even though two weeks have passed [since i thought they would]"

Reply:
Hm, I'd like to keep the pattern on these 2 lines the same - does 
> Even though two weeks had passed, 
>  **not one person had** {believed me, come to see me}

work better for you?

Reply:
I don't really think that fixes my issue. My point is that above, he mentions that Akiha had come to visit so we know he has had visitors since he came to the hospital. This line makes it sound like he has had no visitors which is a weird contradiction.

Reply:
Hm, I think this is actually "two weeks have passed since then" where then is _waking up_, and Akiha's visit was before that.

Reply:
that would indeed make sense